SmartPlaces.Facilities/lib/OntologyMapper/src/OntologyMapper.csproj

```diff
@@ -23,7 +23,7 @@
 	</ItemGroup>
 	
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Azure.DigitalTwins.Parser" Version="6.4.0-preview" />
+		<PackageReference Include="DTDLParser" Version="1.0.4-preview" />
```

Comment:
Any updated consortium eta on a GA'd parser?

Reply:
Week of 2/21